homeassistant/brands/dreo.json

```diff
@@ -0,0 +1,6 @@
+{
+  "domain": "dreo",
```

Comment:
Please remove this file, as the only integration currently supporting the Dreo device is the Dreo integration.
Brands are used if multiple integrations support them.
The fan integration is an entity platform.